core/Archive.php

```diff
@@ -626,12 +642,51 @@ private function get($archiveNames, $archiveDataType, $idSubtable = null)
             }
 
             $resultRow = & $result->get($idSite, $periodStr);
-            $resultRow[$row['name']] = $value;
+
+            if ($chunk->isRecordNameAChunk($row['name'])) {
+                // one combined blob for all subtables
+                $value = unserialize($value);
```

Comment:
I could move all this into another class but would prefer to let it there and refactor it once we do the big archiver refactoring. It does not belong into the Chunks class itself as this class for example doesn't know (and is not supposed to know) that those chunks are serialized in one blob etc.